webpack/scenes/ContentViews/Details/Versions/Delete/RemoveSteps/CVVersionRemoveReview.js

```diff
@@ -58,29 +64,129 @@ const CVVersionRemoveReview = () => {
       {affectedHosts &&
         <>
           <h3>{__('Content hosts')}</h3>
-          <Flex>
-            <FlexItem><ExclamationTriangleIcon /></FlexItem>
-            <FlexItem><p>{__(`${pluralize(hostResponse.length, 'host')} will be moved to content view ${selectedCVNameForHosts} in `)}</p></FlexItem>
-            <FlexItem><Label isTruncated color="purple" href={`/lifecycle_environments/${selectedEnvForHost[0].id}`}>{selectedEnvForHost[0].name}</Label></FlexItem>
-          </Flex>
+          {singleCVHostsCount > 0 && (
+            <Flex>
+              <FlexItem><ExclamationTriangleIcon /></FlexItem>
+              <FlexItem data-testid="single-cv-hosts-remove">
+                <FormattedMessage
+                  id="single-cv-hosts-remove"
+                  defaultMessage="{count, plural, one {{# singular}} other {# {plural}}} will be moved to content view {cvName} in {envName}."
+                  values={{
+                    count: singleCVHostsCount,
+                    singular: __('host'),
+                    plural: __('hosts'),
+                    cvName: selectedCVNameForHosts,
+                    envName: selectedEnvForHost[0] && (
+                      <Label isTruncated color="purple" href={`/lifecycle_environments/${selectedEnvForHost[0].id}`}>
+                        {selectedEnvForHost[0].name}
+                      </Label>
+                    ),
+                  }}
+                />
+              </FlexItem>
+            </Flex>
+          )}
+          {multiCVHostsCount > 0 && (
+            <Flex>
+              <FlexItem><ExclamationTriangleIcon /></FlexItem>
+              <FlexItem>
+                <FormattedMessage
+                  id="multi-cv-hosts-remove"
+                  defaultMessage="Content view {envSingularOrPlural} {envCV} will be removed from {count, plural, one {# {hostSingular}} other {# {hostPlural}}}."
```

Comment:
`plural` isn't defined. I think maybe it needs this?

```suggestion
                  defaultMessage="Content view {envSingularOrPlural} {envCV} will be removed from {count, hostPlural, one {# {hostSingular}} other {# {hostPlural}}}."
```

Reply:
I have updated the formatted message for correct translations. The `plural` keyword is used to handle singular and plural forms based on the `hostCount` value.